cms/static/js/views/active_video_upload_list.js

```diff
@@ -204,27 +227,77 @@ define([
             },
 
             fileUploadDone: function(event, data) {
-                this.setStatus(data.cid, ActiveVideoUpload.STATUS_COMPLETED);
-                this.setProgress(data.cid, 1);
-                if (this.onFileUploadDone) {
-                    this.onFileUploadDone(this.collection);
-                    this.clearSuccessful();
-                }
+                var model = this.collection.get(data.cid),
+                    self = this;
+
+                this.readMessages([
+                    StringUtils.interpolate(
+                        gettext('Upload completed for video {fileName}'),
+                        {fileName: model.get('fileName')}
+                    )
+                ]);
+
+                this.sendStatusUpdate([
+                    {
+                        edxVideoId: model.get('videoId'),
+                        status: 'upload_completed',
+                        message: 'Uploaded completed'
+                    }
+                ]).done(function() {
+                    self.setStatus(data.cid, ActiveVideoUpload.STATUS_COMPLETED);
```

Comment:
@mushtaqak @muhammad-ammar When a video file uploads successfully, and automated processing begins, is the visible status "Pending"? (Is that what you are proposing we change from "In Progress"?)

Also, when does the Status appear as "Uploaded" and when does it appear as "Ready"? We should fix the template sidebar text if necessary.